THIRD-PARTY-LICENSES.properties

```diff
@@ -69,6 +69,29 @@ org.broadinstitute--wdl-biscayne_2.13--84=WDL License https://github.com/openwdl
 org.broadinstitute--wdl-draft2_2.13--84=WDL License https://github.com/openwdl/wdl/blob/master/LICENSE
 org.broadinstitute--wdl-draft3_2.13--84=WDL License https://github.com/openwdl/wdl/blob/master/LICENSE
 
+# delete previous org.broadinstitute group aftera tag after 1.15.0-alpha.13 is created and merged
```

Comment:
FYI, I created a 1.15.0-alpha.13 webservice tag this morning https://github.com/dockstore/dockstore/releases/tag/1.15.0-alpha.13